cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```

Comment:
i'm okay to keep these 2 generated files, could you rename them to `.ts`? We use typescript all over the project.

Reply:
I'm seeing some issues come up with just doing a plain rename. Not just type annotation issues, but like not being able to run the test suite error. It tries to find a .js file but I'm not sure where that's coming from. Perhaps cypress internals. Please advise on best course of action.

Reply:
I would say just delete those files.